src/chapters/functions/intro.rst

```diff
@@ -0,0 +1,98 @@
+Introduction
+============
+
+You have been using functions throughout your learning so far, without receiving a full explanation of how functions work. This chapter focuses explicitly on the details of how functions work, how they can be used, and how you can create functions of your own. 
+
+.. index:: ! function, method
+
+A **function** is a reusable, callable piece of code. Like variables, functions often have names (though we will see in the next chapter that it is possible to create functions without names). 
+
+Here are some functions that you have become familiar with:
+
+- ``console.log``
+- The type conversion functions: ``Number``, ``String``, and ``Boolean``
+- String and array methods, such as ``indexOf``
+
+.. note:: When introducing string and array methods, we noted that a **method** is a function that "belongs to" an object. This distinction is important to keep in mind, and will be explored in depth in a later chapter. For now, note that a method is a special type of function.
+
+Each of the functions we have used works in the same way. Using the function's name, followed by parentheses, we can *call* the function, resulting in some action being carried out. Sometimes, as with ``console.log``, we can provide input data within the parenteses, which the function will use to carry out its action. 
+
+.. admonition:: Example
+
+   The function ``console.log`` carries out the action of printing the provided value(s). In this case, ``"Hello, World!"`` is printed to the console.
+
+   .. sourcecode:: js
+
+      console.log("Hello, World!");
+
+   **Output**
+
+   ::
+
+      Hello, World!   
+   
+In addition to providing input, we may also be given a value in return when calling a function. This is what happened when we have used the type conversion functions, for example. 
+
+.. admonition:: Example
+
+   Type conversion functions *return* a value, that can be used by the calling code. Often, we store the return value of a function in a variable.
```

Comment:
I'm not sure a type conversion function provides the best example for returning a value.  num = Number("46") returns 46, true, but something like arr = "Rutabaga".split('') returns a result that is markedly different from what went into the function.

Reply:
I added an example using `split`.